docs/content/latest/develop/explore-sample-apps.md

```diff
@@ -13,6 +12,8 @@ menu:
     identifier: explore-sample-apps
     parent: develop
     weight: 581
+isTocNested: true
```

Comment:
TOC is still pretty repetitive...

Reply:
At least it's on the side though, I guess...